frontend/src/hooks/pims-api/useApiRequestWrapper.ts

```diff
@@ -6,18 +6,18 @@ import { useDispatch } from 'react-redux';
 import { handleAxiosResponse } from 'utils';
 
 export interface IResponseWrapper<
-  FunctionType extends (...args: any[]) => Promise<AxiosResponse<unknown | undefined>>
+  FunctionType extends (...args: any) => Promise<AxiosResponse<unknown | undefined>>
```

Comment:
@devinleighsmith I did not not find it necessary to make the params an array. Was there a strong reason why this was an any[] before?

Reply:
to me it indicates an expectation of multiple params instead of just one. essentially args is an array of 0 or more elements